### PR TITLE
add a prelude to export commonly used structs, functions, etc

### PR DESCRIPTION
**Pull request by cssivision**
https://github.com/pingcap/raft-rs/issues/45

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -60,3 +60,47 @@ pub use self::status::Status;
...
+        ConfState,
+        ConfChangeType,
+        ConfChange,
+    };
+
+    pub use raft::{Config, Raft};
+
+    pub use storage::{Storage, RaftState};
+
+    pub use raft_log::RaftLog;
```
Comment (siddontang) [inline src/lib.rs]: seem we don't need to export RaftLog.

**Comment by siddontang** [comment]
Thanks @cssivision 

Rest LGTM

PTAL @BusyJay

File: src/lib.rs (hunk 1 above)
Comment (cssivision) [reply src/lib.rs]: fixed

File: src/lib.rs [hunk 2]
```diff
@@ -60,3 +60,45 @@ pub use self::status::Status;
...
+
+    pub use storage::{Storage, RaftState};
+
+    pub use raw_node::{Peer, RawNode, Ready, SnapshotStatus};
+
+    pub use progress::Progress;
+
+    pub use status::Status;
+
+    pub use errors::Error;
```
[the remark this answers is not in the document]
Comment (cssivision) [reply src/lib.rs]: yeah, absolutely.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**